colors/doc/colors.md

```diff
@@ -1,26 +1,25 @@
 ```js script
-import '@divriots/dockit-core/sass-showcases';
-import { html } from '~/md-layout';
+import '@divriots/dockit-core/css-showcases';
 ```
 
 # Colors
 
 ## background-color
 
-```html:html
-<dockit-sass-showcases
+```html story
+<dockit-css-showcases
   css-props-prefix="--color"
   component-class="box"
   style-key="background-color"
-></dockit-sass-showcases>
+></dockit-css-showcases>
```

Comment:
since I fixed `dockit-css-showcases` to properly show opacity in the documentation context, it became clear that one of the colors you have is transparent... not sure if you meant that or created such color by mistake, please double check

![image](https://user-images.githubusercontent.com/137844/151864517-f6adb9d1-3a67-4d17-a823-d501e83121f0.png)